README.md

```diff
@@ -1,70 +1,185 @@
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## 🛠️ Built With <a name="built-with"></a>
 
-### `npm test`
+### Tech Stack <a name="tech-stack"></a>
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+<details>
+  <summary>🖥️ Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+    <li><a href="https://reactrouter.com/">React Router</a></li>
+    <li><a href="https://axios-http.com/">Axios</a></li>
+    <li><a href="https://tailwindcss.com/">Tailwind CSS</a></li> 
+  </ul>
+</details>
 
-### `npm run build`
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+<!-- FEATURES -->
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### 🔑 Key Features <a name="key-features"></a>
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+### Home Page
 
-### `npm run eject`
+- Upon loading, the webapp displays a list of anime categories, each with corresponding numeric values fetched from the API.
+- Users can click on a category item to navigate to the details page for that category.
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+### Details Page
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+- The details page shows comprehensive information about the selected anime category.
+- A "Back" button allows users to return to the home page.
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+<!-- LIVE DEMO -->
 
-## Learn More
+## 🚀 Demo <a name="live-demo"></a>
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+- [🔗 Live Demo Link](https://kit0-0.github.io/react-metrics-anime/)
+- [🔗 Presentation link](https://drive.google.com/file/d/1FQD9Mj7v21qBXtykPSL5ooh5xHmrh7q5/view?usp=sharing)
 
```

Comment:
- [ ] Excellent work on articulating your project with a high level of clarity and professionalism. I appreciate the effort you've put into your presentation. However, I would like to kindly request that you re-record the video to align with our presentation requirements.

  - One of the specific requirements outlined is to highlight a particularly significant piece of code that you take pride in. Regrettably, this aspect was not fully met in your current video presentation.

   - To ensure that we adhere to the project's standards and enhance its overall quality, I would encourage you to review the detailed project requirements provided below. By addressing this specific aspect, we can ensure that your presentation fully encompasses the intended objectives.
<img width="1025" alt="Screenshot 2023-08-07 at 22 35 09" src="https://github.com/kit0-0/react-metrics-anime/assets/107192076/890dc27c-8dc8-4f07-a451-a1892420ec44">